package.json

```diff
@@ -118,7 +118,7 @@
     "fix:eslint": "npm run check:eslint -- --fix",
     "prepare": "node etc/prepare.js",
     "preview:docs": "ts-node etc/docs/preview.ts",
-    "release": "standard-version -i HISTORY.md",
+    "release": "standard-version a -i HISTORY.md",
```

Comment:
Needs `-a`
and we chatted a bit offline: we might want a script here that asserts the repo has no other outstanding staged files as this flag could accidentally commit source changes. I would say add a `git reset` here but that could be destructive so something that would just exit failure if any staged changes are outside the docs folder would cover us.